akka-multi-node-testkit/src/main/scala/akka/remote/testkit/MultiNodeSpec.scala

```diff
@@ -160,29 +160,41 @@ object MultiNodeSpec {
   require(selfName != "", "multinode.host must not be empty")
 
   /**
-   * Port number of this node. Defaults to 0 which means a random port.
+   * TCP Port number to be used when running tests on TCP. 0 means a random port.
    *
    * {{{
    * -Dmultinode.port=0
    * }}}
    */
-  val selfPort: Int = Integer.getInteger("multinode.port", 0)
+  val tcpPort: Int = Integer.getInteger("multinode.port", 0)
 
-  require(selfPort >= 0 && selfPort < 65535, "multinode.port is out of bounds: " + selfPort)
+  require(tcpPort >= 0 && tcpPort < 65535, "multinode.port is out of bounds: " + tcpPort)
```

Comment:
@patriknw, I removed the `tcp` in the property to make it backward compatible. 

The `selfPort` will default to `tcpPort` unless explicit requests to be `udpPort`. 

I think this give a good level of backward compatibility since new behavior needs to be explicitly enabled.